Assets/AppCenter/Editor/AppCenterPostBuild.cs

```diff
@@ -429,7 +444,12 @@ private static void OnPostprocessInfo(PlistDocumentWrapper info, AppCenterSettin
                 setStringMethod.Invoke(urlType, new object[] { "CFBundleTypeRole", "None" });
                 setStringMethod.Invoke(urlType, new object[] { "CFBundleURLName", ApplicationIdHelper.GetApplicationId() });
                 var urlSchemes = urlType.GetType().GetMethod("CreateArray").Invoke(urlType, new[] { "CFBundleURLSchemes" });
-                urlSchemes.GetType().GetMethod("AddString").Invoke(urlSchemes, new[] { "appcenter-" + settings.iOSAppSecret });
+
+                // Add custom schemes defined in Unity players settings.
+                foreach (var scheme in schemes)
+                {
+                    urlSchemes.GetType().GetMethod("AddString").Invoke(urlSchemes, new[] { scheme });
```

Comment:
Why don't we have a null check here as well?

Reply:
this is a previous implementation that has been moved to a loop and it had no the null check